pyomo/contrib/mindtpy/mip_solve.py

```diff
@@ -320,18 +276,14 @@ def handle_main_infeasible(main_mip, solve_data, config):
 
 
 def handle_main_max_timelimit(main_mip, main_mip_results, solve_data, config):
-    """
-    This function handles the result of the latest iteration of solving the MIP problem given that solving the
+    """This function handles the result of the latest iteration of solving the MIP problem given that solving the
     MIP takes too long.
 
-    Parameters
-    ----------
-    main_mip: Pyomo model
-        the MIP main problem
-    solve_data: MindtPy Data Container
-        data container that holds solve-instance data
-    config: ConfigBlock
-        contains the specific configurations for the algorithm
+    Args:
+        main_mip (Pyomo model): the MIP main problem.
+        main_mip_results (SolverResults): results from solving the MIP main subproblem.
+        solve_data (MindtPySolveData): data container that holds solve-instance data.
+        config (ConfigBlock): the specific configurations for MindtPy.
     """
     # TODO check that status is actually ok and everything is feasible
```

Comment:
Do we still need this TODO?

Reply:
We didn't check if the MIP solver has found a feasible solution. Is it necessary here? If not, we can remove this TODO.

Reply:
The issue here is that the maxtime limit has been reached, so we pull up the results that there are, if we have found a valid feasible solution, we take that, if not, we can at least use the lower bound? This is more involved than what I thought.

Reply:
Got it.